docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM phusion/baseimage:0.10.1
+LABEL maintainer "chevdor@gmail.com"
+
+RUN apt-get update && \
+	apt-get upgrade -y && \
+	apt-get install -y cmake pkg-config libssl-dev && \
+	curl https://sh.rustup.rs -sSf | sh && \
+	rustup update nightly && \
```

Comment:
i was getting error the following error when trying to run `rustup` as it wasn't added to the PATH.

i fixed it by adding the following lines of code after the `curl` command:

```bash
RUN echo 'PATH="$/root/.cargo/bin:$PATH";' >> ~/.bash_profile && . ~/.bash_profile && . /root/.cargo/env
ENV PATH "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/root/.cargo/bin"
RUN which rustup && which cargo
```

Reply:
FYI, the alternative to running `ENV PATH "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/root/.cargo/bin"` is the following:

```bash
RUN echo 'PATH="$/root/.cargo/bin:$PATH";' >> ~/.bash_profile && . ~/.bash_profile && . /root/.cargo/env
```